client/plots/wsiviewer/defaults.ts

```diff
@@ -1,5 +1,5 @@
 import Settings from './Settings'
-import { copyMerge } from '#rx'
+import { copyMerge } from '../../rx'
```

Comment:
Please revert this like so it's:
`import { copyMerge } from '#rx'`

Reply:
@aacic, I reverted this change.